cassandra/cluster.py

```diff
@@ -1775,6 +1777,9 @@ def connect(self, keyspace=None, wait_for_all_pools=False):
                     self.shutdown()
                     raise
 
+                # Update the information about tablet support after connection handshake.
+                self.load_balancing_policy._tablets_routing_v1 = self.control_connection._tablets_routing_v1
```

Comment:
Isn't it necessary to set `_tablets_routing_v1` also on child policies? This is what `populate()` also did. Maybe the code doesn't work properly if the set policy is some policy that has token aware policy as child - the child policy (token aware policy) won't have `_tablets_routing_v1` set?

Reply:
That's true, do you think that creating another function only to populate `_tablets_routing_v1` will be a good solution? Or should I investigate more why calling `populate` was causing issues and return to calling it?

Reply:
I think it's a good idea to investigate why populate was causing issues.